doc/openapi.yaml

```diff
@@ -0,0 +1,217 @@
+          $ref: '#/components/responses/200LintingSuccessful'
+
+  /lintocsp:
+    post:
+      summary: Lints an OCSP response
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+  /linttbsocsp:
+    post:
+      summary: Creates and lints an OCSP response from the specified ResponseData
+      requestBody:
+        $ref: '#/components/requestBodies/LintRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/200LintingSuccessful'
+          
+  /profiles:
+    get:
+      summary: Retrieves the profiles available for linting
+      responses:
+        '200':
+          description: A list of profiles available for linting
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/LintProfile'
+                  
+  /linters:
+    get:
+      summary: Retrieves the linters available for linting
+      responses:
+        '200':
+          description: A list of linters available for linting
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Linter'
+
+components:
+  requestBodies:
+    LintRequestBody:
+      description: The parameters for the linting request
+      required: true
+      content:
+        application/x-www-form-urlencoded:
+          schema:
+            $ref: '#/components/schemas/LintRequest'
+
+  responses:
+    200LintingSuccessful:
+      description: The response for the specified linting request
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/LintResponse'
+        text/html:
+          schema:
+            type: string
+        text/plain:
+          schema:
+            type: string
+
+  schemas:
+    FindingSeverity:
+      type: string
+      enum:
+        - meta
+        - debug
+        - info
+        - notice
+        - warning
+        - error
+        - bug
+        - fatal
+      description: The severity of a linter finding
+
+    ResponseFormat:
+      type: string
+      enum:
+        - json
+        - html
+        - text
+      description: The response format
+      default: json
+
+    LintRequest:
+      type: object
+      required:
+        - b64input
+      properties:
+        b64input:
+          type: string
+          description: The Base64 or PEM-encoded input
+        format:
+          $ref: '#/components/schemas/ResponseFormat'
+        profile:
+          type: string
+          description: The name of a profile
+          default: autodetect
+        severity:
+          $ref: '#/components/schemas/FindingSeverity'
+
+    LintResponse:
+        type: array
+        items:
+          $ref: '#/components/schemas/LintFinding'
+
+    LintFinding:
```

Comment:
I'm wondering if `Lint` is necessary as a prefix here. Are there other kinds of findings we'd be returning here? If not, `Finding` or `Lint` could be simpler and less redundant.

Reply:
I'm partial to the current naming, as the object represents a finding reported through the execution of a linter. That being said, I can change if others agree with the name change.

Reply:
I guess my question is, what other type of finding is expected from a Meta Linter?

Reply:
Also, LintFinding is part of LintResponse so it's quite redundant. 

If you've dealt with autogenerated clients, it's going to be a ugly set of models for most languages. Sure we could use an overlay but that's not widely supported for OAPI, so it's not a practical solution to this either

Reply:
These are good points. On the flip side, this object has a "Finding" field, so I find it strange that a "Finding" object would have a "Finding" field. To me, that creates confusion as to what a "finding" is: is it a simple string, or is it an object?

Reply:
The Finding field could be called "Description" instead as well to resolve that. But I agree the naming here is tricky.

Reply:
I'm going to resolve this conversation and proceed with merging this PR.  @sigmavirus24: If you still think changes are required, please start a new discussion at https://github.com/pkimetal/pkimetal/discussions/new?category=general.